libs/wingsdk/src/shared-aws/bucket.inflight.ts

```diff
@@ -410,6 +413,21 @@ export class BucketClient implements IAwsBucketClient {
         });
         break;
       case BucketSignedUrlAction.UPLOAD:
+        if (opts?.uploadId !== undefined) {
```

Comment:
How is this related to signed urls?

Reply:
You may generate a signed url specifically to upload a part of a multipart upload. When there's a multipart upload object attached to the sign request options then we know you want a signed url for a multipart upload.